src/braintree/getBraintreeJsUrls.ts

```diff
@@ -12,14 +12,16 @@ export function getBraintreeJsUrls(version?: string): IBraintreeUrls {
         FASTLANE_JS: fastlaneJs,
         DATA_COLLECTOR_JS: dataCollectorJs,
         GOOGLE_JS_URL: googleJsUrl,
-        JS_VERSION: jsVersion
+        JS_VERSION: jsVersion,
+        PAYPAL_CHECKOUT_URL: paypalCheckoutUrl,
     } = braintreeConstants;
     version ??= jsVersion;
     const clientJsURL = `${base}/${version}/${clientJs}`;
     const appleJsURL = `${base}/${version}/${appleJs}`;
     const braintreeGoogleJsURL = `${base}/${version}/${googleJs}`;
     const dataCollectorJsURL = `${base}/${version}/${dataCollectorJs}`;
     const fastlaneJsURL = `${base}/${version}/${fastlaneJs}`;
+    const paypalCheckoutURL = `${base}/${version}/${paypalCheckoutUrl}`;
```

Comment:
Having two variable names that only differ in their case, seems like asking for trouble. My two bits.

Reply:
Fixed